ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/site/js/accordion.js

```diff
@@ -0,0 +1,346 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+(function() {
+
+    class Accordion extends FormView.FormPanel {
+
+        static NS = FormView.Constants.NS;
+        static IS = "accordion";
+        static bemBlock = 'cmp-accordion'
+        static selectors  = {
+            self: "[data-" + this.NS + '-is="' + this.IS + '"]',
+            item: `.${Accordion.bemBlock}__item`,
+            button: `.${Accordion.bemBlock}__button`,
+            panel: `.${Accordion.bemBlock}__panel`
+        };
+
+        static cssClasses = {
+            button: {
+                disabled: "cmp-accordion__button--disabled",
+                expanded: "cmp-accordion__button--expanded"
+            },
+            panel: {
+                hidden: "cmp-accordion__panel--hidden",
```

Comment:
this is not needed

Reply:
removed all the duplicate code with sites, added sites library as dependency.